### ISSUE TITLE
BalochiDictionary - When an entry is displayed it should display from right-to-left

### ISSUE THREAD

**Issue by peteroatley** [sillsdev/webonary#642]
In the Search and Browse when entries are in a list they display correctly right-to-left. But when an individual entry is displayed it displays left-to-right.

<img width="260" alt="image" src="https://github.com/sillsdev/webonary/assets/144733976/82ce961c-f5ab-46f6-b635-b686674615d6">

<img width="450" alt="image" src="https://github.com/sillsdev/webonary/assets/144733976/5234046e-3001-4922-8efc-2695843a48b2">

**Comment by stevendyk** [comment]
Ron Lockwood requested we move this to High Priority on 24-03-15. @stevendyk can demonstrate the issue.

**Comment by peteroatley** [comment]
Phil's comments from Slack: I'm not able to reproduce this problem. Perhaps it would help if there were a few links in the issue to the affected pages. 
Also, it would help if there was an example for what it should look like, since I can't read the script.

Judging from the screen shots, it looks like both examples are the same, as far as the order of the characters, so the issue isn't actually LTR vs RTL. My guess would be they are complaining because the second example is aligned on the left margin rather than the right one.

**Comment by peteroatley** [comment]
@phillip-hopper you are correct that the Urdu text is RTL and that it is the text justification which is different.

Here is a link to another entry یات (yát) :
https://www.webonary.org/balochidictionary/browse/browse-vernacular/?key=bcc&letter=%DB%8C

And the full entry for the yát:
https://www.webonary.org/balochidictionary/g9b891fc6-939d-4d10-a4c0-5611e4642b77/

Searching for yát produces a right-justified result.
https://www.webonary.org/balochidictionary?s=y%C3%A1t&search=Search&key=bcc-Latn-x-com&pos=&semantic_domain=&search_options_set=1&match_whole_words=1

Here is a breakdown of the Urdu letters in yát:
https://unicode.scarfboy.com/?s=%DB%8C%D8%A7%D8%AA

yát is Balochi latin script if you are doing a search.
